pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework.js

```diff
@@ -67,44 +67,31 @@ router.get('/', function (req, res, next) {
       assessment.renderText(
         res.locals.assessment,
         res.locals.urlPrefix,
-        function (err, assessment_text_templated) {
+        async function (err, assessment_text_templated) {
```

Comment:
Might be an issue to pass an async function as a callback, like Nathan mentioned earlier.

Reply:
@nwalters512 Would it be possible to rewrite the enclosing `router.get()` callback function in this file to be entirely async? I'm not sure how to do so because most of these methods in `assessment.js` do not have async versions.

If it's not possible, would we use the `async` library with `async.series()`, or should we leave this as is?